src/inference/inference.ts

```diff
@@ -140,41 +170,31 @@ export class Inference {
           ' documents. See https://docs.pinecone.io/models for supported models.'
       );
     }
-    // Destructure `options` with defaults
-    // Note: If the user passes in key:value pairs in `options` that are not the following, they are ignored
+
     const {
       topN = documents.length,
       returnDocuments = true,
-      rankFields = ['text'],
       parameters = {},
+      rankFields = undefined,
```

Comment:
Do we need to destructure rankFields and set to undefined? Isn't it undefined by default? I think I don't entirely understand why we can't keep the `rankFields = ['text']` default

Reply:
Ah good call, good call